benchmarks/benchmarks/sparse_linalg_lobpcg.py

```diff
@@ -23,13 +23,13 @@ def _sakurai(n):
         Appl. Num. Anal. Comp. Math. Vol. 1 No. 2 (2004) """
 
     A = scipy.sparse.eye(n, n)
-    d0 = array(r_[5,6*ones(n-2),5])
-    d1 = -4*ones(n)
+    d0 = array(r_[5, 6 * ones(n - 2), 5])
+    d1 = -4 * ones(n)
     d2 = ones(n)
-    B = scipy.sparse.spdiags([d2,d1,d0,d1,d2],[-2,-1,0,1,2],n,n)
+    B = scipy.sparse.spdiags([d2, d1, d0, d1, d2], [-2, -1, 0, 1, 2], n, n)
 
-    k = arange(1,n+1)
-    w_ex = sort(1./(16.*pow(cos(0.5*k*pi/(n+1)),4)))  # exact eigenvalues
+    k = arange(1, n + 1)
+    w_ex = sort(1. / (16. * pow(cos(0.5 * k * pi / (n + 1)), 4)))  # exact eigenvalues
```

Comment:
This doesn't look right, too many spaces. This is the case in multiple places. Please have a look at what PEP8 says about expressions with operators:

    If operators with different priorities are used, consider adding whitespace
    around the operators with the lowest priority(ies)

See also the example at https://www.python.org/dev/peps/pep-0008/#other-recommendations

Can you please correct this whole PR for that?